examples/raw_input/RawInput.cpp

```diff
@@ -0,0 +1,85 @@
+////////////////////////////////////////////////////////////
+// Headers
+////////////////////////////////////////////////////////////
+#include <SFML/Graphics.hpp>
+
+#include <vector>
+
+
+namespace
+{
+std::filesystem::path resourcesDir()
+{
+#ifdef SFML_SYSTEM_IOS
+    return "";
+#else
+    return "resources";
+#endif
+}
+} // namespace
+
+
+////////////////////////////////////////////////////////////
+/// Entry point of application
+///
+/// \return Application exit code
+///
+////////////////////////////////////////////////////////////
+int main()
+{
+    // Create the main window
+    sf::RenderWindow window(sf::VideoMode({800u, 600u}), "SFML Raw Mouse Input", sf::Style::Titlebar | sf::Style::Close);
+    window.setVerticalSyncEnabled(true);
+
+    // Load the application font and pass it to the Effect class
+    const auto font = sf::Font::loadFromFile(resourcesDir() / "tuffy.ttf").value();
+
+    auto vec2ToString = [](const auto& vec2)
```

Comment:
```suggestion
    const auto vec2ToString = [](const sf::Vector2i& vec2)
```
There's no need for this function to be generic. We're only stringifiying this one template specialization. We can also mark this `const`. Might also be worth moving this closer to the two callsites. If we're going to use lambdas then we might as well minimize the scope of the lambda.